frontend/src/services/getUsers.ts

```diff
@@ -0,0 +1,19 @@
+const getUsers = async (url: string) => {
```

Comment:
this should be renamed into apiService for example
Also keep the base url here, and create request url inside the function by params, so the methods outside this file are not aware about any server details